modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -55,157 +60,261 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
 the changes from the codemod as a single isolated commit (separate from other changes) so you can
 roll back more easily if necessary.
 
-## Component Deprecations
+## Deprecations and Removals
+
+#### Deprecations
+
+We add the [@deprecated](https://jsdoc.app/tags-deprecated.html) from JSDoc to code that we plan to deprecate in the near future.
+Al though you can still consume this code, we want consumers to move to a utility or component that is more stable.
+
+#### Removals
+
+Removals are deletions from our codebase and you can no longer consume this component. We've either promoted or replaced a component or utility.
+
+### Drawer - _Removal_
```

Comment:
@mannycarrera4 i think the suggestion was to have them be in sections like
### Deprecations
#### useThemeRTL
etc

### Removals
#### Drawer
etc

and this would also be reflected at the top (line 16)

Reply:
ah, ok, updating!